apps/api/src/sibyl/config.py

```
@@ -123,7 +123,9 @@ def validate_security_settings(self) -> "Settings":
                     "CRITICAL: Default PostgreSQL password 'sibyl_dev' is forbidden in production. "
                     "Set SIBYL_POSTGRES_PASSWORD to a secure value."
                 )
-            if self.store == "surreal" and self.resolved_surreal_url.startswith("memory://"):
+            if requires_surreal_support(
```

Ruff (0.15.12)

[error] Undefined name `requires_surreal_support`

(F821)